driver/driver_test.go

```diff
@@ -213,6 +215,10 @@ func (f *fakeStorageDriver) GetVolume(ctx context.Context, id string) (*godo.Vol
 }
 
 func (f *fakeStorageDriver) CreateVolume(ctx context.Context, req *godo.VolumeCreateRequest) (*godo.Volume, *godo.Response, error) {
+	if f.createVolumeErr != nil && f.createVolumeErrResponse != nil {
+		return nil, f.createVolumeErrResponse, f.createVolumeErr
```

Comment:
Should we return as soon as one of `f.createVolumeErr` or `f.createVolumeErrResponse` is non-nil (as opposed to requiring both)?

Reply:
Sure! It gives more flexibility in the tests. Added.